packages/landing/app/layout.tsx

```diff
@@ -47,18 +45,9 @@ export default function RootLayout({
   return (
     <html lang="en" className={stabilGrotesk.variable}>
       <body className="font-sans bg-surface-25 text-em-high">
-        <Script
-          src="https://cdn.usefathom.com/script.js"
-          data-auto
-          data-site={siteId}
-          data-spa="hash"
-          defer
-        />
-
-        <Providers>
-          <Navbar />
-          {children}
-        </Providers>
+        <Fathom />
```

Comment:
In this case I would prefer to call it `FathomAnalytics`  just to be clear it's analytics.